tutorials/beginners-cookbook/index.md

```diff
@@ -306,47 +307,74 @@ var countyData=ee.FeatureCollection('TIGER/2018/Counties');
 var countyConnect=countyData.filter(ee.Filter.eq('STATEFP', '09'));
 // Get the union of all the county geometries in Connecticut.
 var countyConnectDiss=countyConnect.union();
-// Add the layer to the map with a specified color and layer name.
-Map.addLayer(countyConnectDiss, {color: 'red'}, 'Chicago dissolved');
-// Find the rectangle that emcompasses the southernmost, westernmost,
-// easternmost, and northernmost points of the feature.
+//Create a circular area using the first county in the Connecticut  FeatureCollection.
+var circle=ee.Feature(countyConnect.first()).geometry().centroid().buffer(50000)
```

Comment:
Please add single blank space around assignments (" = ") here and throughout, and end all lines with ";".

Reply:
Done